definitions/npm/@apollo/client_v3.x.x/flow_v0.142.x-/client_v3.x.x.js

```diff
@@ -70,13 +70,24 @@ declare interface GraphQL$ExecutionResult<
   data?: TData | null;
   extensions?: TExtensions;
 }
-
-// @apollo/client/link/core/types.d.ts
-// Globals to support global ApolloLink class
-declare type ApolloClient$Path = $ReadOnlyArray<string | number>;
+declare type GraphQL$VariableNode = Object;
```

Comment:
Shouldn't use `Object` as this resolves to `any`. Use `{ ...  }` or `{ [key: string]: any }` instead

Reply:
Fixed in 93a521257c7eb1afd802f07067c1b16e20cbbfa4